### PR TITLE
GovernanceExecuteProcessReporterTest

### LINKED ISSUE #10969: Add unit test for shardingsphere-governance
Hi community,

This issue is for #10887.

## Aim
Add unit test for shardingsphere-governance project to test its public functions.

## Basic Qualifications
- Java  
- Maven  
- Junit.Test

## Classes

Add unit test for classes to test its public functions.

- [ ] GovernanceExecuteProcessReporter

## Examples FYI
- GovernanceTransactionContextsTest

### PR DESCRIPTION
**Pull request by SriSatyaT**
Fixes #10969 

Changes proposed in this pull request:
-
-Added Unit Test for Governance Execute Process Reporter

### CODE REVIEW

**Comment by menghaoranss** [comment]
@SriSatyaT ci build failed.

**Comment by SriSatyaT** [comment]
@menghaoranss Will look into it.

**Comment by SriSatyaT** [comment]
Thanks @menghaoranss. Will look into it.

**Comment by menghaoranss** [comment]
> Thanks @menghaoranss. Will look into it.

May be you should pull the master branch code first.

**Comment by tristaZero** [comment]
Hi @SriSatyaT Is this your first PR in this community? Please have my welcome! BTW, could I know what makes you come here?

**Comment by SriSatyaT** [comment]
@tristaZero Yes it's my first pr. I have introduced in slack channel 😃.

File: shardingsphere-governance/shardingsphere-governance-context/src/test/java/org/apache/shardingsphere/governance/context/Process/GovernanceExecuteProcessReporterTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    public void assertReportWithLogicSQL() {
+        ExecuteProcessReporter executeProcessReporter = new GovernanceExecuteProcessReporter(shardingSphereEventBus);
+        ExecutionGroupContext executionGroupContext = new ExecutionGroupContext(ExecuteProcessConstants.EXECUTE_PROCESS_TYPE, "ds_0", "schema_0", "table_0", "sql_0");
+        ExecuteProcessReportEvent executeProcessReportEvent = new ExecuteProcessReportEvent(executeProcessContext, executionGroupContext);
+        executeProcessReporter.report(executeProcessReportEvent);
+        verify(shardingSphereEventBus).post(any(ExecuteProcessSummaryReportEvent.class));
+        verify(shardingSphereEventBus).post(any(ExecuteProcessUnitReportEvent.class));
+        verify(shardingSphereEventBus).post(any(LogicSQL.class));
+    }
+
```
Comment (menghaoranss) [inline shardingsphere-governance/shardingsphere-governance-context/src/test/java/org/apache/shardingsphere/governance/context/Process/GovernanceExecuteProcessReporterTest.java:73]: Missing `}` , and please keep a blank line at the end of file

**Review by menghaoranss — CHANGES_REQUESTED** [review]

**Comment by menghaoranss** [comment]
@SriSatyaT hi, there are some minor problems that cause to ci build failed , and I have explained in the `requested changes`, please check again.

**Comment by SriSatyaT** [comment]
@menghaoranss Thank you for reviewing pr. I'll modify them.

**Comment by tristaZero** [comment]
@SriSatyaT Hi Any block?

**Comment by SriSatyaT** [comment]
> @SriSatyaT Hi Any block?

Hey, Yeah I have to recheck it. I might did many mistakes while making a pr.

**Comment by tristaZero** [comment]
@SriSatyaT No hurries and worries, go ahead. ;-）

**Comment by SriSatyaT** [comment]
> @SriSatyaT No hurries and worries, go ahead. ;-）

Sure. Thank you :)

**Comment by menghaoranss** [comment]
@SriSatyaT hi, This PR has been for a long time. If there is no progress, I will close this PR later. If you are interested in continuing, please feel free to commit another pr, thanks.

**Comment by SriSatyaT** [comment]
> @SriSatyaT hi, This PR has been for a long time. If there is no progress, I will close this PR later. If you are interested in continuing, please feel free to commit another pr, thanks.

Alright. I'll try pick some other issues.

**Pull request closed without merging**